godtools/App/Features/Tools/SwiftUI/AllTools/ItemsList.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  ItemsList.swift
+//  godtools
+//
+//  Created by Levi Eggert on 5/12/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import SwiftUI
+import UIKit
+
+struct ItemsList<ListItem: Identifiable, ListItemView: View>: View {
```

Comment:
I'm wondering how this is going to work in adding the Spotlight and Categories sections.  I was going to put them all in a List/LazyVStack together, but this only allows a single type of view?